src/main/resources/init_dbscheme

```diff
@@ -79,6 +79,7 @@ CREATE TABLE `internet_shop`.`users`
     `username` varchar(256) NOT NULL,
     `login`    varchar(256) NOT NULL,
     `password` varchar(256) NOT NULL,
+    `salt` BIT(16) NULL DEFAULT NULL,
```

Comment:
every user must have salt, so it can't be null

Reply:
agree